src/worldofzuul/Trump.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package worldofzuul;
+
+/**
+ *
+ * @author Kasper
+ */
+public class Trump extends Character {
+    /**
+     * The name of the character
+     */
+    private String name;
+    
+    public Trump(){
```

Comment:
Should also have a docblock

Reply:
`Trump.java` could be written as:
```java
public class Trump extends Character {
  private String name = "Donald Trump";
}
```

Reply:
Fixed in f6966ef8ef659f6045729267b7289b226522252e